app/views/admin/users/_user_invitation_status.haml

```diff
@@ -1,15 +1,36 @@
-- if user.invitation_status == :active
-  - if user.access_locked?
-    Account locked.
+- if user.inactive?
```

Comment:
Should this be two separate views at this point? (One for active users table, one for inactive users table)

Reply:
That sounds like a good plan now.  Thanks!

Reply:
Ok, done!